filters/general/filter_55_1Hosts_Pro/configuration.json

```
@@ -0,0 +1,20 @@
+{
+    "name": "1Hosts (Pro)",
+    "description": "List for blocking pesky ads, trackers, and malware. Strict version: has some minimal breakages, prioritizes privacy & safety (adblocking) over UX.",
```

It breaks Facebook by blocking `graph.facebook.com`. Not minimal breakage:)
```suggestion
    "description": "List for blocking pesky ads, trackers, and malware. Strict version: has some minimal breakages, prioritizes privacy & safety (adblocking) over UX e.g blocks graph.facebook.com.",
```